examples/medplum-patient-intake-demo/src/App.tsx

```diff
@@ -23,7 +25,10 @@ export function App(): JSX.Element | null {
       menus={[
         {
           title: 'Charts',
-          links: [{ icon: <IconUser />, label: 'Patients', href: '/Patient' }],
+          links: [
+            { icon: <IconUser />, label: 'Patients', href: '/Patient' },
+            { icon: <IconUser />, label: 'Customize intake form', href: '/customize-form' },
```

Comment:
Nitpick suggestion: As this new page isn't a chart, I'd place it in a different menu group.

Reply:
Thanks, moved to a separate group and changed the icon to a more appropriate one